test/controllers/sections/guarantor/GuarantorVatControllerSpec.scala

```diff
@@ -153,7 +153,7 @@ class GuarantorVatControllerSpec extends SpecBase with MockUserAnswersService {
 
         status(result) mustEqual SEE_OTHER
         redirectLocation(result).value mustEqual
-          controllers.sections.guarantor.routes.GuarantorArrangerController.onPageLoad(testErn, testLrn, NormalMode).url
+          controllers.sections.guarantor.routes.GuarantorIndexController.onPageLoad(testErn, testLrn).url
```

Comment:
The test name says redirect to 'arranger controller' I think this should be 'index controller'